arangodb-http/src/main/scala/io/funkode/arangodb/http/ArangoClientHttp.scala

```diff
@@ -286,15 +298,15 @@ object ArangoClientSchema:
 
   val schemaEncoderForHttp: HttpEncoder[Schema] = new HttpEncoder[Schema]:
     override def encode[R](r: R)(using S: Schema[R]) =
-      Body.fromChunk(zio.schema.codec.JsonCodec.encoderFor(S).encode(r))
+      Body.fromCharSequence(zio.schema.codec.JsonCodec.jsonEncoder(S).encodeJson(r))
```

Comment:
I also found this option:
```scala
val bodyStream: Stream[Throwable, Char] = JsonEncoder[R].encodeJsonStream(r)
Body.fromStream(bodyStream)

Reply:
Let's change this in another PR. There were some issues when I tried it